### PR TITLE
YUNIKORN-8: Pull application and container history from core

### PR DESCRIPTION
**Pull request by akhilpb001**

### CODE REVIEW

**Comment by akhilpb001** [comment]
@yangwwei @wilfred-s  @sunilgovind 
Could you please review the PR

File: src/app/components/cluster-info/cluster-info.component.ts [hunk 1]
```diff
@@ -59,8 +61,13 @@ export class ClusterInfoComponent implements OnInit {
         this.updateContainerStatusData(data);
       });
 
-    this.jobHistoryData = this.generateHistoryData();
-    this.containerHistoryData = this.generateHistoryData();
+    this.scheduler.fetchAppHistory().subscribe(data => {
+      this.jobHistoryData = this.getAreaChartData(data);
```
Comment (wilfred-s) [inline src/app/components/cluster-info/cluster-info.component.ts]: Can we rename this to app history data? 
Or should we do it similarly to what we did in core/shim and change from job to app in a separate jira?

Comment (yangwwei) [reply src/app/components/cluster-info/cluster-info.component.ts]: I think we can open a separate JIRA to track that, as that is not blocking the release.

**Comment by yangwwei** [comment]
I have tested this locally, it works well with https://github.com/apache/incubator-yunikorn-core/pull/109. Let's get this merged. Thanks a lot, @akhilpb001 .

**Comment by yangwwei** [comment]
Just one question, since the backend will return # of running apps/containers. Is there a way to indicate the chart is showing running ones?
In the future, can we include total ones, and completed ones in this chart too if possible?

**Pull request merged**